infra/base-images/base-runner/bad_build_check

```diff
@@ -433,6 +460,10 @@ function main {
   result=$?
   checks_failed=$(( $checks_failed + $result ))
 
+  check_reproduce_empty_file $FUZZER
```

Comment:
Can you just add a env var check here (false by default). The env var can be called `MORE_BAD_BUILD_CHECKS` or somethign similar.

Reply:
I added the env var check and also disable the check for `c++` targets.